navigation/libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/utils/time/TimeUtils.java

```diff
@@ -0,0 +1,65 @@
+package com.mapbox.services.android.navigation.v5.utils.time;
+
+import java.util.Calendar;
+import java.util.Locale;
+import java.util.concurrent.TimeUnit;
+
+public class TimeUtils {
+
+  private static final String DURATION_ILLEGAL_ARGUMENT = "Duration must be greater than zero.";
+  private static final String ARRIVAL_TIME_STRING_FORMAT = "%tl:%tM %tp%n";
+  private static final int STRING_BUILDER_CAPACITY = 64;
+  private static final String DAYS = " days ";
```

Comment:
Can we localize these strings?

Reply:
Not given the way the current util class is used - we can figure out a better way that would allow localization